starter/src/templates/dev.tsx

```diff
@@ -95,23 +97,36 @@ export const getPath: GetPath<TemplateProps> = ({ document }) => {
 };
 
 const Dev: Template<TemplateRenderProps> = (props) => {
+  const [themeMode, setThemeMode]= React.useState<boolean>(false);
   const { document } = props;
   const entityFields = devTemplateStream.stream.schema
     .fields as unknown as YextSchemaField[];
 
   return (
-    <VisualEditorProvider
-      document={document}
-      entityFields={entityFields}
-      tailwindConfig={tailwindConfig}
-    >
-      <Editor
-        document={document}
-        componentRegistry={componentRegistry}
-        themeConfig={themeConfig}
-        localDev={true}
-      />
-    </VisualEditorProvider>
+      <div>
+        <div className={"flex-container"}>
+          <ChakraButton className={"toggle-button"} onClick={() => {
```

Comment:
Can we just use the same button component that clear local changes uses? We don't need to import a whole new library just for a button. It doesn't even look like a button in your screenshot, just text.

Reply:
With real local dev idk if it will be more complicated to get dev mode working where we moved the theme mgr button, but I'd rather not move this into visual-editor for now. Just use a normal html button and slap some styling on it so it doesn't look horrendous.